src/main/java/Duke.java

```diff
@@ -1,10 +1,193 @@
+                + "list - Lists all tasks recorded." + System.lineSeparator()
+                + "done <task number> - Marks selected task number as done with an X." + System.lineSeparator()
+                + "bye - Exits the taskbot." + System.lineSeparator()
+                + "____________________________________________________________";
+        System.out.println(helpMessage);
+    }
+
+    public static void printAddedTodo(String todoDescription) {
+        String addedMessage = "____________________________________________________________" + System.lineSeparator()
+                + "Alright! I've successfully added this task:" + System.lineSeparator()
+                + "[T]" + "[ " + "] " + todoDescription + System.lineSeparator()
+                + "You now have " + tasksCounter + " tasks in the list!" + System.lineSeparator()
+                + "____________________________________________________________" + System.lineSeparator();
+        System.out.println(addedMessage);
+    }
+
+    public static void printAddedDeadline(String deadlineDescription, String deadlineBy) {
+        String addedMessage = "____________________________________________________________" + System.lineSeparator()
+                + "Alright! I've successfully added this task:" + System.lineSeparator()
+                + "[D]" + "[ " + "] " + deadlineDescription + "(by: " + deadlineBy + ")" + System.lineSeparator()
+                + "You now have " + tasksCounter + " tasks in the list!" + System.lineSeparator()
+                + "____________________________________________________________" + System.lineSeparator();
+        System.out.println(addedMessage);
+    }
+
+    public static void printAddedEvent(String eventDescription, String eventAt) {
+        String addedMessage = "____________________________________________________________" + System.lineSeparator()
+                + "Alright! I've successfully added this task:"  + System.lineSeparator()
+                + "[E]" + "[ " + "] " + eventDescription + "(at: " + eventAt + ")" + System.lineSeparator()
+                + "You now have " + tasksCounter + " tasks in the list!" + System.lineSeparator()
+                + "____________________________________________________________" + System.lineSeparator();
+        System.out.println(addedMessage);
+    }
+
+    public static void printList() {
+        System.out.println("___________________________________________________________" + System.lineSeparator()
+                + "As requested, here are the tasks in your list:");
+        if (tasksCounter == 0) {
+            System.out.println("There are no tasks recorded!");
+        }
+        for (int i = 0; i < tasksCounter; i++) {
+            String taskType = tasks[i].getType();
+            String byOrAt = "";
+
+            if (taskType.equals("D")) {
+                byOrAt = "by: ";
+            }
+            else if (taskType.equals("E")) {
```

Comment:
For your "else" and "else if" statements, it should be in this "} else if {" format, where both braces are to be in the same line (based on the java coding standard given to us).